src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -78,14 +129,16 @@
    :allow-authorization-extensions   schema/Bool
    :allow-duplicate-certs            schema/Bool
    :allow-subject-alt-names          schema/Bool
+   :allow-auto-renewal               schema/Bool
+   :auto-renewal-cert-ttl            TTLDuration
    :autosign                         AutoSignInput
    :cacert                           schema/Str
    :cadir                            schema/Str
    :cacrl                            schema/Str
    :cakey                            schema/Str
    :capub                            schema/Str
    :ca-name                          schema/Str
-   :ca-ttl                           schema/Int
+   :ca-ttl                           TTLDuration
```

Comment:
I think this should remain an Int throughout, and we should convert it from a string to an int when we construct the settings.